sefaria/model/topic.py

```diff
@@ -69,6 +70,11 @@ def _validate(self):
         super(Topic, self)._validate()
         if getattr(self, 'subclass', False):
             assert self.subclass in self.subclass_map, f"Field `subclass` set to {self.subclass} which is not one of the valid subclass keys in `Topic.subclass_map`. Valid keys are {', '.join(self.subclass_map.keys())}"
+        if getattr(self, 'image', False):
```

Comment:
Its possible this code isnt doing what you expect it to do. 
Youre using the index() method to test for the existence of a substring and expecting a custom exception to be thrown. 
However, according to the docs, a `ValueError` is what is thrown in such a case. 
1. To correct this block of code, you'd need to catch that error and then raise the custom exception. That a lot of lines of code.  I'd suggest re-writing to use `in` or `find()` so you dont have to handle the ValueError explicitly. 
2. Is there code outside this function that is catching the custom exception for it to be useful? (for example, the Topic.save() method?)

Reply:
Ah OK thanks Ephraim, I will revisit this :)

Reply:
Hi Ephraim - pushing a commit to fix this now, and to prefer the use of `in` in the context of an `assert`. In my prior conversation with @nsantacruz I recall him mentioning possibly us preferring the custom exceptions over `assert` statements, so I just want to loop him in here to check and clarify. Thanks!

Reply:
A further update (based also on our conversation) -- removed any need for an `assert` by using the new validation pattern.